src/client/src/models/Scene.js

```diff
@@ -69,6 +75,14 @@ const Scene = {
   setCurrent: function(sceneId) {
     const list = Scene.getScenes()
     Scene.current = list[sceneId]
+  },
+
+  setInputFormat: function(format) {
+    const list = JSON.parse(localStorage.getItem('input-format'))
+    list['inputFormat'] = format
```

Comment:
So I'm wondering about this section.

What is the type of the value returned by `JSON.parse(localStorage.getItem('input-format')) `?
You called the variable "list" which makes me think that the variable is an array, but then you assign to a property as if it is an object.

So I'm not sure if you are doing what you expect, or if we just need to rename "list" to something less misleading.